### ISSUE TITLE
emqx crashed

### ISSUE THREAD

**Issue by SpringHgui** [emqx/emqx#9197]
### What would you like to be added or enhanced?

```
500 INTERNAL_ERROR:error, function_clause, [{emqx_mgmt_api_nodes,format,['emqx@127.0.0.1',{error,{'EXIT',{{badmatch,{error,enoent}},[{emqx_vm,read_otp_version,0,[{file,"emqx_vm.erl"},{line,419}]},{emqx_mgmt,otp_rel,0,[{file,"emqx_mgmt.erl"},{line,533}]},{emqx_mgmt,node_info,0,[{file,"emqx_mgmt.erl"},{line,131}]}]}}}],[{file,"emqx_mgmt_api_nodes.erl"},{line,290}]},{emqx_mgmt_api_nodes,'-list_nodes/1-lc$^0/1-0-',1,[{file,"emqx_mgmt_api_nodes.erl"},{line,260}]},{emqx_mgmt_api_nodes,list_nodes,1,[{file,"emqx_mgmt_api_nodes.erl"},{line,260}]},{minirest_handler,apply_callback,3,[{file,"minirest_handler.erl"},{line,111}]},{minirest_handler,handle,2,[{file,"minirest_handler.erl"},{line,44}]},{minirest_handler,init,2,[{file,"minirest_handler.erl"},{line,27}]},{cowboy_handler,execute,2,[{file,"cowboy_handler.erl"},{line,41}]},{cowboy_stream_h,execute,3,[{file,"cowboy_stream_h.erl"},{line,318}]},{cowboy_stream_h,request_process,3,[{file,"cowboy_stream_h.erl"},{line,302}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]
```

```
[root@svrhz10-6 ~]# cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"


[root@svrhz10-6 ~]# emqx_ctl broker
sysdescr  : EMQX
version   : 5.0.8
datetime  : 2022-10-21T14:01:39.701802706+08:00
uptime    : 9 minutes, 24 seconds
```

### Why is this needed?

不要挂

**Comment by SpringHgui** [comment]
![image](https://user-images.githubusercontent.com/22363718/197124564-236940b4-6183-4c0a-9d1e-60f06f451408.png)

**Comment by zhongwencool** [comment]
Fix in v5.0.16

**Comment by IoTThinks** [comment]
Seem have the same issue on Ubuntu with emqx 5.0.18
NAME="Ubuntu"
VERSION="20.04.5 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.5 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

When one node is down, the alert appears.
![image](https://user-images.githubusercontent.com/29994971/221567092-f9259637-b463-412d-a844-30352577e3fd.png)
